src/index.css

```diff
@@ -1 +1,68 @@
 
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;800&display=swap');
+@import url("https://waldo-ui.netlify.app/styles/main.css");
+
+
+@import url(pages/homepage/styles.css);
+
+*,
+::before,
+::after {
+    box-sizing: border-box;
+    margin: 0px;
+    padding: 0px;
+}
+
+body{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    position: relative;
+    color: var(--primary-bg);
+    background-color: var(--primary-clr);
+    font-family: 'Open Sans', sans-serif;
+}
+
+li{
+    list-style: none;
+}
+
+a{
+    text-decoration: none;
+}

```

Comment:
try to avoid generic styling to element give it a class then style it

Reply:
Thanks for the suggestions.